chrome/content/zotero-better-bibtex/schomd.coffee

```diff
@@ -132,8 +208,10 @@ Zotero.BetterBibTeX.schomd.bibliography = (citekeys, {style, libraryID} = {}) ->
   itemIDs = @itemIDs(citekeys, {libraryID})
   return '' if itemIDs.length == 0
 
-  url = "http://www.zotero.org/styles/#{style ? 'apa'}"
-  style = Zotero.Styles.get(url)
+  urlstyle = "#{style ? 'apa'}"
+  style = Zotero.Styles.get("http://www.zotero.org/styles/#{urlstyle}")
```

Comment:
extremely minor point, but I think good for maintainability later: wouldn't `styleurl` disclose the intent of the variable more closely than `urlstyle`?